src/agent/WMICommunication.h

```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <memory>
+#include <vector>
+#include <Wbemidl.h>
+
+
+class WMICommunication
+{
+public:
+	~WMICommunication();
+
+
+	bool WMIInit();
+
+	bool GetSMARTDataViaWMI();
+
+	std::shared_ptr<IWbemLocator> initialLocatorToWMI;
```

Comment:
please mark members with some prefix. It is way easier to read code when you can easily see which variable is local one and which is a member). At this moment I use m_ prefix, but we can discuss this

Reply:
m_ will be fine.